src/mon/MonCommands.h

```diff
@@ -869,6 +869,31 @@ COMMAND("osd pool get-quota " \
         "name=pool,type=CephPoolname ",
         "obtain object or byte limits for pool",
         "osd", "r", "cli,rest")
+COMMAND("osd pool application enable " \
```

Comment:
At the CDM, we discussed updating "ceph fs new", the built-in RGW pool creation, and adding new rbd CLI for creating pools to avoid having users pass around magic strings.